src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -218,6 +211,171 @@ namespace N
                 "Delete [using System.Collections;]@22");
         }
 
+        [Fact]
+        public void Using_Delete_ChangesCodeMeaning()
```

Comment:
New tests need WorkItem for the linked issue. Not sure about updated tests.

Reply:
Really? I've only ever done that for bugs.

Reply:
@davidwengier I thought this is considered a bug :smile:
If it's more like a feature request implementation, probably no need.